src/SFA.DAS.ApplyService.Web/Views/Users/ExistingAccount.cshtml

```diff
@@ -23,6 +22,9 @@
                     <span class="govuk-error-message" asp-validation-for="FirstTimeSignin"></span>
```

Comment:
I think this needs to go below 'the heading'.  So move to line 28, and move everything else down a line?

Reply:
Done